portfolio/src/main/webapp/script.js

```diff
@@ -68,27 +68,52 @@ async function getUserImages(){
     });
 }
 
+let map;
+
+let editMarker;
+
 function createMap() {
- // Lattitude and longitude of Chavez Ravine.
-  var chavezRavine = {lat: 34.073851, lng: -118.242147};
-  const map = new google.maps.Map(
+  // Lattitude and longitude of the USA.
```

Comment:
Fix typo for latitude when you have the chance.

Reply:
Fixed.